runtime/src/main/scala/akka/grpc/internal/RequestBuilderImpl.scala

```diff
@@ -333,6 +383,14 @@ final class ScalaBidirectionalStreamingRequestBuilder[I, O](
 
   override def withHeaders(headers: MetadataImpl): ScalaBidirectionalStreamingRequestBuilder[I, O] =
     new ScalaBidirectionalStreamingRequestBuilder[I, O](descriptor, channel, defaultOptions, settings, headers)
+
+  override def setDeadline(deadline: Duration): ScalaBidirectionalStreamingRequestBuilder[I, O] =
+    new ScalaBidirectionalStreamingRequestBuilder[I, O](
+      descriptor,
+      channel,
+      defaultOptions.withDeadlineAfter(deadline.toMillis, TimeUnit.MILLISECONDS),
```

Comment:
see my comment about FiniteDuration, this will probably fail otherwise for some non finite durations